modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/css/main.scss

```diff
@@ -162,20 +162,59 @@
 		}
 	}
 
+	h3.component-title {
+		font-size: $h3-font-size;
+
+		@include media-breakpoint-down(sm) {
+			font-size: $h3-font-size-mobile;
+		}
+	}
+
+	.message-user-display {
+		margin-bottom: 0.5rem;
+		margin-top: 0.5rem;
+	}
+
+	.social-interaction {
+		display: flex;
+		margin-left: -0.5rem;
+
+		> div {
+			margin-right: 0.5rem;
+		}
+	}
+
 	@include media-breakpoint-up(sm) {
+		.col-md-4 {
+			.message-user-display {
+				width: 100%;
+			}
+
+			.social-interaction {
+				margin-left: 0;
+				position: static;
+				right: 0;
+				top: 0.25rem;
+				width: 100%;
+
+				> div {
+					float: left;
+				}
+			}
+		}
+
```

Comment:
I can confirm changing the position to static addresses the issue, but I am not sure col-md-4 (I also tried col-md-3) is the correct scope.

Reply:
`col-md-4` is correct, should be 33.33333% width.